config/routes.rb

```diff
@@ -2,6 +2,8 @@
   root to: "pages#home"
 
   match "/", to: "paws_up#create", as: "paws_ups", via: :post
+  post '/cycle/create', to: 'cycle#create', as: 'create_cycle'
```

Comment:
small nit: canonical create is `post '/cycles', to: 'cycles#create', as: 'cycles'`

Reply:
also maybe I prefer:

```ruby
resources :cycles, only: [:create]
```